### ISSUE TITLE
Cmder: bad link on website

### ISSUE THREAD

**Issue by davidrans** [sqshq/sampler#133]
The website has a link to a sketchy looking site (`https://cmder.net/`) instead of https://cmder.app/

**Comment by WFANG12719** [comment]
Yes, that's a wrong link making me confused until I got here. Anyway, I went to the cmder.app finally.
